packages/spektate/src/IDeployment.ts

```diff
@@ -433,12 +434,19 @@ export const status = (deployment: IDeployment): string => {
       deployment.hldToManifestBuild.result === "canceled")
   ) {
     return "canceled";
+  } else if ((deployment.srcToDockerBuild &&
```

Comment:
probably cleaner to do `else if ((deployment. srcToDockerBuild?.result  ?? deployment.dockerToHldReleaseStage?result ?? deployment.hldToManifestBuild?.result) === "success")`